-- Fix the PEP 621 license fields so that scikit-bio builds with the version of
-- setuptools available in FreeBSD Ports.
--- pyproject.toml.orig	2026-08-03 09:55:00 UTC
+++ pyproject.toml	2026-08-03 09:55:00 UTC
@@ -8,8 +8,7 @@
 readme = "README.rst"
 requires-python = ">=3.10"
-license = "BSD-3-Clause"
-license-files = ["LICENSE.txt"]
+license = {text = "BSD-3-Clause"}
 authors = [
     {name = "scikit-bio development team", email = "qiyunzhu@gmail.com"}
